RiotSwiftUI/Modules/Authentication/Login/Coordinator/AuthenticationLoginCoordinator.swift

```diff
@@ -29,7 +29,7 @@ enum AuthenticationLoginCoordinatorResult {
     /// Continue using the supplied SSO provider.
     case continueWithSSO(SSOIdentityProvider)
     /// Login was successful with the associated session created.
-    case success(session: MXSession, password: String)
+    case success(session: MXSession, password: String?)
```

Comment:
Rather than altering this and adding the fallback VC presentation stuff below, do you think we could reuse the one in the AuthCoordinator by adding a `fallback` case to the `AuthenticationLoginCoordinatorResult` to request it is shown?